src/main/resources/db/migration/V2024_04_01_1647__CIV-11301_trial_arrangements_claimant_scenario.sql

```diff
@@ -0,0 +1,32 @@
+/**
+ * Add scenario
+ */
+INSERT INTO dbs.scenario (name, notifications_to_delete, notifications_to_create)
+VALUES ('Scenario.AAA6.CP.Trial.Arrangements.Required.Claimant',
```

Comment:
We must use the ticket scenario if it is "bothParties". The same for the notifications, because if we save them in DB with a different name than the one indicated in the ticket we lose traceability to delete them in future tickets.

In my branch I have made the necessary changes to be able to do it this way without having any errors. https://github.com/hmcts/civil-service/pull/4411/files